app/build.gradle

```diff
@@ -115,19 +115,17 @@ dependencies {
     implementation project(':library:model-events')
     implementation project(':library:model-loader')
     implementation project(':library:sync')
-
     implementation project(':ui:account')
     implementation project(':ui:base')
     implementation project(':ui:base-app')
     implementation project(':ui:everystudent')
     implementation project(':ui:shortcuts')
+    implementation project(':ui:articles-aem-renderer')
     implementation project(':ui:tract-renderer')
-
```

Comment:
guessing that you used the Android Studio dependency editor, it tends to strip out extra whitespace :)

I keep the whitespace gaps to help group related dependencies together for organization and ease of seeing what's included.

I always alpha sort dependencies within each section